packages/grid_client/tests/utils.ts

```diff
@@ -54,4 +55,29 @@ async function returnRelay() {
   return relay;
 }
 
-export { log, generateHash, generateInt, splitIP, bytesToGB, RemoteRun, returnRelay };
+async function k8sWait(masterIP, masterName, workerName, waitTime, newWorkerName?) {
```

Comment:
can we pick a better name instead of `masterIp`? cause it's an object from the ssh client

Reply:
Names were updated in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/390/commits/25e34483bdfa54279036a1d157ae4b50480f66b3